apps/editing-toolkit/package.json

```diff
@@ -33,7 +33,7 @@
 		"build:wpcom-block-editor-nav-sidebar": "calypso-build --env source='wpcom-block-editor-nav-sidebar'",
 		"build:wpcom-block-editor-nux": "calypso-build --env source='wpcom-block-editor-nux'",
 		"build:wpcom-site-editor-nux": "calypso-build --env source='wpcom-site-editor'",
-		"clean": "npx rimraf dist 'editing-toolkit-plugin/*/dist'",
```

Comment:
`|| true` is required in case the glob expansion doesn't find any file (https://github.com/yarnpkg/berry/blob/master/CHANGELOG.md#shell)